### PR TITLE
AC New Automated Test - LRAC-11513 | master

### PR DESCRIPTION
**Pull request by joshitagaki**
https://issues.liferay.com/browse/LRAC-11513
[Testray](https://testray.liferay.com/home/-/testray/case_results?testrayBuildId=1953711137)
Let me know what you think about the name for the macro and path file. ACTests looks better than ACABTest and _Tests_ is what we use in AC for the AB tests. Open to suggestions on renaming the files if you feel it necessary

### CODE REVIEW

**Comment by joshitagaki** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACTests.macro [hunk 1]
```diff
@@ -0,0 +1,31 @@
+definition {
+
+	macro deleteABTest {
+		var key_testName = "${testName}";
+
+		Click(locator1 = "ACTests#AB_TEST_NAME");
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACTests.macro:6]: Hey @joshitagaki, I think you could use:

`ACNavigation.openItem(itemName = "${testName}");`

File: portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACTests.macro [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+	macro statusABTest {
+		AssertTextEquals(
+			locator1 = "ACTests#LABEL_STATUS",
+			value1 = "${status}");
+	}
+
+	macro terminateABTest {
+		var key_testName = "${testName}";
+
+		Click(locator1 = "ACTests#AB_TEST_NAME");
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACTests.macro:24]: Hey @joshitagaki, I think you could use:

`ACNavigation.openItem(itemName = "${testName}");`

What do you think?

File: portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACTests.macro (hunk 1 above)
Comment (joshitagaki) [reply portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACTests.macro:6]: Oh nice yes that would work. Thanks!

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/ABTest.testcase [hunk 3]
```diff
@@ -84,4 +84,72 @@ definition {
...
+		}
+
+		task ("Go to Tests") {
+			ACNavigation.goToTests();
+		}
+
+		task ("Assert AB test Deleted") {
+			AssertElementNotPresent(
+				key_testName = "${nameABTest}",
+				locator1 = "ACTests#AB_TEST_NAME");
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/ABTest.testcase:151]: Hey @joshitagaki, I think here you could validate that the empty state message is appearing, that way we could use the whole test using macro and not put functions directly in the .testcase. What do you think?

`ACUtils.viewGenericText(textValueList = "There are no tests found.");`

**Comment by joshitagaki** [comment]
ci:test:sf

**Review by ViniLopes20 — APPROVED** [review]
Very good test and thanks for these new macros and paths. LGTM 🎉

**Review by christianstokes — APPROVED** [review]
Looks great, thank you @joshitagaki 

And I agree with using ACTests.macro as the name, it makes sense to me, since that is the main heading used in the product. (Similar to ACSegments, for example)

**Comment by christianstokes** [comment]
ci:forward

**Pull request closed without merging**